### PR TITLE
Heirloom spears

### PR DESCRIPTION
**Pull request by Muparadzi**
Updated the following spears, loosely following the pattern below:-

+1 (+1 speed)
+2 (+1 damage)
+3 (+1 speed)

Simple pike
Iron Pitchfork 
Pitchfork 
Wide Leaf Spear
Fine Steel Hewing Spear
Fine Pike
Hoplite Spear

### CODE REVIEW

**Review by namidaka — CHANGES_REQUESTED** [review]
Keep on the good work

**Comment by Muparadzi** [comment]
2h spears completed, continuing with 1h now

File: items.json [hunk 1]
```diff
@@ -6183,10 +6183,10 @@
...
     "type": "Polearm",
-    "price": 5437,
-    "weight": 2.35,
+    "price": 10500,
+    "weight": 2.25,
     "heirloomLevel": 3,
-    "tier": 5.83303261,
+    "tier": 8.516016,
     "requirement": 0,
     "flags": [],
```
Comment (namidaka) [inline items.json:6191]: A little bit weak. Consider adding 1 damage or 1 speed.

File: items.json [hunk 2]
```diff
@@ -43489,10 +43413,86 @@
...
+  {
+    "id": "crpg_fine_steel_spear_h3",
+    "name": "Fine Steel Spear +3",
+    "culture": "Empire",
+    "type": "Polearm",
+    "price": 10363,
+    "weight": 1.83,
+    "heirloomLevel": 3,
+    "tier": 8.453262,
+    "requirement": 0,
```
Comment (namidaka) [inline items.json:43470]: Same here +1 damage or speed

File: items.json [hunk 3]
```diff
@@ -58872,10 +58872,10 @@
...
     "type": "Polearm",
-    "price": 2297,
-    "weight": 0.87,
+    "price": 2547,
+    "weight": 0.8,
     "heirloomLevel": 3,
-    "tier": 3.4408195,
+    "tier": 3.67510152,
     "requirement": 0,
     "flags": [],
```
Comment (namidaka) [inline items.json:58880]: Original Tier was 5.81 , this one probably requires a considerable buff

File: items.json [hunk 4]
```diff
@@ -65339,10 +65339,10 @@
...
     "culture": "Aserai",
     "type": "Polearm",
-    "price": 2411,
-    "weight": 1.72,
+    "price": 4854,
+    "weight": 1.53,
     "heirloomLevel": 3,
-    "tier": 3.54928064,
+    "tier": 5.45453167,
     "requirement": 0,
```
Comment (namidaka) [inline items.json:65346]: consider adding one stats on this one too

File: items.json [hunk 5]
```diff
@@ -76523,10 +76523,10 @@
...
     "type": "Polearm",
-    "price": 9434,
-    "weight": 1.97,
+    "price": 10229,
+    "weight": 1.88,
     "heirloomLevel": 1,
-    "tier": 8.016542,
+    "tier": 8.391522,
     "requirement": 0,
     "flags": [],
```
Comment (namidaka) [inline items.json:76531]: original tier is 9.7 , h1 is a little bit too weak. Try to get at least tier 9

File: items.json [hunk 6]
```diff
@@ -76599,10 +76599,10 @@
...
     "type": "Polearm",
-    "price": 5290,
-    "weight": 1.97,
+    "price": 11701,
+    "weight": 1.79,
     "heirloomLevel": 3,
-    "tier": 5.73965549,
+    "tier": 9.049611,
     "requirement": 0,
     "flags": [],
```
Comment (namidaka) [inline items.json:76607]: Try adding one stat. I'd rather have it tier 10.1 (going slightly over tier 10) then losing 0.7 tier

File: items.json [hunk 7]
```diff
@@ -77922,10 +77922,10 @@
...
     "culture": "Aserai",
     "type": "Polearm",
-    "price": 9497,
-    "weight": 2.07,
+    "price": 10965,
+    "weight": 1.96,
     "heirloomLevel": 1,
-    "tier": 8.046889,
+    "tier": 8.72598,
     "requirement": 0,
```
Comment (namidaka) [inline items.json:77929]: opriginal tier was 9.74 , it's a bit too weak. Try not to get heirloomed version further than 1 tier. 0.8/0.7 is acceptable

File: items.json [hunk 8]
```diff
@@ -78084,10 +78084,10 @@
...
     "culture": "Aserai",
     "type": "Polearm",
-    "price": 5324,
-    "weight": 2.07,
+    "price": 11892,
+    "weight": 1.84,
     "heirloomLevel": 3,
-    "tier": 5.761384,
+    "tier": 9.132158,
     "requirement": 0,
```
Comment (namidaka) [inline items.json:78091]: If you have to choose between 9.1 and 10.1 , i'd rather go slightly overboard

File: items.json [hunk 9]
```diff
@@ -82456,10 +82456,10 @@
...
     "type": "Polearm",
-    "price": 4889,
-    "weight": 2.38,
+    "price": 10226,
+    "weight": 2.14,
     "heirloomLevel": 3,
-    "tier": 5.477475,
+    "tier": 8.390356,
     "requirement": 0,
     "flags": [],
```
Comment (namidaka) [inline items.json:82464]: same here .

File: items.json [hunk 10]
```diff
@@ -91181,10 +91181,10 @@
...
     "culture": "Khuzait",
     "type": "Polearm",
-    "price": 3113,
-    "weight": 2.17,
+    "price": 6359,
+    "weight": 1.95,
     "heirloomLevel": 3,
-    "tier": 4.166421,
+    "tier": 6.392777,
     "requirement": 0,
```
Comment (namidaka) [inline items.json:91188]: same here, original tier was 7.04

File: items.json [hunk 11]
```diff
@@ -98446,10 +98446,10 @@
...
     "type": "Polearm",
-    "price": 5401,
-    "weight": 3.35,
+    "price": 10944,
+    "weight": 3.13,
     "heirloomLevel": 3,
-    "tier": 5.81020832,
+    "tier": 8.716494,
     "requirement": 0,
     "flags": [],
```
Comment (namidaka) [inline items.json:98454]: same here , original tier was 9.8

**Review by namidaka — CHANGES_REQUESTED** [review]

**Comment by Muparadzi** [comment]
I have addressed comments regarding tier of items

**Review by namidaka — APPROVED** [review]

**Pull request merged**